applications/asset_tracker_v2/doc/ui_module.rst

```diff
@@ -7,7 +7,7 @@ User Interface module
    :local:
    :depth: 2
 
-The User Interface module controls and monitors the UI elements on the nRF9160 DK and Thingy:91.
+The User Interface module controls and monitors the UI elements on nRF91 Series Development kits and Thingy:91.
```

Comment:
```suggestion
The User Interface module controls and monitors the UI elements on nRF91 Series development kits and Thingy:91.
```
I think using either "development kits", "Development Kits" or "DKs" is fine here, but "Development kits" is not usually a capitalization we have in the documentation.